_studio/mfx_lib/shared/include/mfx_h264_enc_common_hw.h

```diff
@@ -842,29 +860,66 @@ namespace MfxHwH264Encode
         }
 
         template <typename T> friend mfxExtBufferProxy GetExtBuffer(const T & par);
+        template <typename T> friend mfxExtBufferProxy GetExtBuffer(const T & par, mfxU32 fieldId);
+        template <typename T> friend mfxExtBufferProxy GetExtBufferFEI(const T & bs);
```

Comment:
no sense in special functions for FEI, they do exactly the same things

Reply:
GetExtBufferFEI required, because it consume pointer to mfxBitstream, so different way for field access (need use ->)
We should use different function names for recognize how work with first argument